Library/ViewModels/ManagePledgeViewModelTests.swift

```diff
@@ -446,6 +446,42 @@ internal final class ManagePledgeViewModelTests: TestCase {
       self.showActionSheetMenuWithOptions.assertValues([[.viewRewards]])
     }
   }
+  
+  func testMenuButtonTapped_WhenProject_IsPledgeOverTime_doesNotInclude_chooseAnotherReward() {
+    let project = Project.template
+      |> Project.lens.state .~ .live
+
+    let mockService = MockService(
+      fetchManagePledgeViewBackingResult: .success(.template),
+      fetchProjectResult: .success(project),
+      fetchProjectRewardsResult: .success([.template])
+    )
+    
+    let mockConfigClient = MockRemoteConfigClient()
+    mockConfigClient.features = [
+      RemoteConfigFeature.pledgeOverTime.rawValue: true
```

Comment:
I agree with Amy - just add the `featureNoShippingAtCheckout` flag here, too, so that the test makes sense!

Reply:
Also agree – the test has been updated to include the `featureNoShippingAtCheckout` flag. Thanks for the input!